### ISSUE TITLE
Release Twisted 11.0

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4980]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)| @glyph reported|
|-|-|
|Trac ID|trac#4980|
|Type|enhancement|
|Created|2011-03-22 22:37:15Z|

The best release of Twisted yet.

<details><summary>Searchable metadata</summary>

```
trac-id__4980 4980
type__enhancement enhancement
reporter__glyph glyph
priority__normal normal
milestone__Twisted_11_0 Twisted-11.0
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1300833435000000 1300833435000000
changetime__1301856919000000 1301856919000000
version__None None
owner__jesstess jesstess
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [31395]) Move the _c_urlarg removal to the removals section in core, the misc section in all affected subprojects

refs [#4980](https://github.com/twisted/twisted/issues/4980)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra commented|
|-|-|

Some of the sphinx doc changes (mostly merged by lvh i think) are mentioned in the changelog, some aren't. We should be consistent and mark all of them as .doc or all of them as .misc.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

We shouldn't mention them.  People reading the NEWS file don't care about each separate change we made.  Feel free to delete them (make sure you get the duplicates across different NEWS files).

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra commented|
|-|-|

(In [31412]) remove references to lore2sphinx tickets from Documentation section, refs [#4980](https://github.com/twisted/twisted/issues/4980)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra commented|
|-|-|

Replying to [exarkun](#note_3):
> We shouldn't mention them.  People reading the NEWS file don't care about each separate change we made.  Feel free to delete them (make sure you get the duplicates across different NEWS files).

Removed them and checked the other news files as well. Was only one more that mentioned a lore2sphinx change but this was made in 10.2 so no need to remove it now.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun commented|
|-|-|

> > (make sure you get the duplicates across different NEWS files)
> Removed them and checked the other news files as well.

Did you check the top-level `NEWS` file?

Also, these tickets should still appear in the _Other_ section.

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_8"></a>|@glyph commented|
|-|-|

Looking pretty good, but just waiting on some [build results](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/releases/release-11.0.0-4980) to make sure the `setup.py` changes don't cause anything untoward to happen...

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)|@glyph set owner to @glyph<br>@glyph set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_10"></a>|@glyph set owner to @jesstess<br>@glyph set status to `new`|
|-|-|

Looks good.  Land.

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_11"></a>|@jesstess set status to `closed`|
|-|-|

Release Twisted 11.0.0.

Author: jesstess

Reviewers: glyph, exarkun

Fixes: [#4980](https://github.com/twisted/twisted/issues/4980)
